net.gradator.gtkatlantic.yaml

```diff
@@ -0,0 +1,24 @@
+app-id: net.gradator.gtkatlantic
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: gtkatlantic
+rename-icon: gtkatlantic
+rename-desktop-file: gtkatlantic.desktop
+finish-args:
```

Comment:
Needs `--share=network`, otherwise fails to list servers and doesn't allow to create a new game.